src/libstd/sys/windows/fs.rs

```diff
@@ -310,6 +313,49 @@ impl File {
         }
     }
 
+    #[cfg(target_os = "uwp")]
+    pub fn file_attr(&self) -> io::Result<FileAttr> {
+        unsafe {
+            let mut info: c::FILE_BASIC_INFO = mem::zeroed();
+            let size = mem::size_of_val(&info);
+            cvt(c::GetFileInformationByHandleEx(self.handle.raw(),
```

Comment:
Is this function available off the uwp target? Or is it only available on the uwp target?

Reply:
According to https://docs.microsoft.com/en-us/windows/desktop/api/winbase/nf-winbase-getfileinformationbyhandleex#requirements it's supported down to vista